src/optim_problems/run_opt.jl

```diff
@@ -59,23 +58,22 @@ what you can add to the opt_config:
 function run_opt(ts_data::ClustData,
                     opt_data::OptDataCEP,
                     opt_config::Dict{String,Any},
-                    fixed_design_variables::Dict{String,OptVariable};
-                    solver::Any=CbcSolver(),
+                    fixed_design_variables::Dict{String,Any},
+                    optimizer::DataType;
                     lost_el_load_cost::Number=Inf,
-                    lost_CO2_emission_cost::Number=Inf,
-                    k_ids::Array{Int64,1}=Array{Int64,1}())
+                    lost_CO2_emission_cost::Number=Inf)
   # Create dictionary for lost_load_cost of the single elements
   lost_load_cost=Dict{String,Number}("el"=>lost_el_load_cost)
   # Create dictionary for lost_emission_cost of the single elements
   lost_emission_cost=Dict{String,Number}("CO2"=>lost_CO2_emission_cost)
   # Add the fixed_design_variables and new setting for slack costs to the existing config
   set_opt_config_cep!(opt_config;fixed_design_variables=fixed_design_variables, lost_load_cost=lost_load_cost, lost_emission_cost=lost_emission_cost)
 
-  return run_opt(ts_data,opt_data,opt_config;solver=solver,k_ids=k_ids)
+  return run_opt(ts_data,opt_data,opt_config,optimizer)
 end
 
 """
-     run_opt(ts_data::ClustData,opt_data::OptDataCEP,fixed_design_variables::Dict{String,OptVariable};solver::Any=CbcSolver(),descriptor::String="",co2_limit::Number=Inf,lost_el_load_cost::Number=Inf,lost_CO2_emission_cost::Number=Inf,existing_infrastructure::Bool=false,intrastorage::Bool=false)
+     run_opt(ts_data::ClustData,opt_data::OptDataCEP,optimizer::DataTyple;descriptor::String="",co2_limit::Number=Inf,lost_el_load_cost::Number=Inf,lost_CO2_emission_cost::Number=Inf,existing_infrastructure::Bool=false,limit_infrastructure::Bool=false,storage::String="none",transmission::Bool=false,print_flag::Bool=true,print_level::Int64=0)
```

Comment:
storage as simple String, not split up Bool

Reply:
Sounds good.